pkg/apis/core/types.go

```diff
@@ -3677,6 +3677,22 @@ const (
 	SupplementalGroupsPolicyStrict SupplementalGroupsPolicy = "Strict"
 )
 
+// PodSELinuxChangePolicy defines how the container's SELinux label is applied to all volumes used by the Pod.
+type PodSELinuxChangePolicy string
+
+const (
+	// Recursive relabeling of all Pod volumes by the container runtime.
```

Comment:
I think these comments do not end up in the generated api docs, so it would be better to put these in the comment for the SELinuxChangePolicy field.

Reply:
Added. It's getting bigger and bigger :-)